proto/meta.proto

```diff
@@ -304,7 +305,7 @@ message UpdateWorkerNodeSchedulabilityResponse {
 }
 
 message ListAllNodesRequest {
-  common.WorkerType worker_type = 1;
+  optional common.WorkerType worker_type = 1;
```

Comment:
This is warned as a breaking change. But this RPC is only used by risectl and this PR, so it's fine.